src/views/v2/day/type-separator.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * View: Day View Type separator
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/day/type-separator.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+use Tribe\Events\Views\V2\Utils;
```

Comment:
* :no_entry_sign: There must be exactly one blank line after the file comment
* :no_entry_sign: Missing @package tag in file comment
* :no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: ec10acf96a57868375ad13c6673d44b1</sub>